media-gfx/xpaint/xpaint-2.10.2-r1.ebuild

```diff
@@ -42,6 +42,11 @@ BDEPEND="
 	virtual/pkgconfig
 "
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-libtool-clang.patch
```

Comment:
Put the version you based this on in the filename

Reply:
I had it but I removed it because I am using the same patch bot for *2.10.2-r1* and *3.1.3*.  Is it ok to patch *3.1.3* with a patch named *2.10.2*?

Reply:
Yeah, that's no problem. I name it after the first version in tree it applies to/the one I based it on.